aws_lambda_powertools/utilities/parser/models/kinesis_firehose.py

```diff
@@ -9,15 +8,15 @@
 class KinesisFirehoseRecordMetadata(BaseModel):
     shardId: str
     partitionKey: str
-    approximateArrivalTimestamp: str
+    approximateArrivalTimestamp: int
```

Comment:
Even in [documentation](https://docs.aws.amazon.com/lambda/latest/dg/services-kinesisfirehose.html) the field is a string, I created again a kinesis -> kinesis firehose -> lambda and in the service payload this is not a string, this a timestamp field.

![image](https://user-images.githubusercontent.com/4295173/194336065-616fd97b-0c8e-4559-be71-6aaa3d196735.png)

I'll open a ticket to the documentation team to fix that.